### DISCUSSION TITLE
How to run/debug RP2 (dev) from IDE (VSCodium)

### DISCUSSION THREAD

**Discussion by haavardf** [eprbell/rp2#131] [Q&A]
I am new to Python and VSCodium. Not new to programing but I am sure I would save time by using some of your knowlege and time :-) and others might come after me and find this helpfull.

I have followed the deveopers guide to get started. And I can run the project using 'make run'. To use my configuration I just went in to the makefile, and changed the parameters there. But I would like to run it from the IDE, so I can also run it in debug mode.

Best practise tips are welcome.

Thank you for the fast and good support I have recived so far.

**Comment by eprbell** [comment]
I'm old-school and debug with printf, so I haven't used the integrated debugger. I see much documentation online though: e.g. https://code.visualstudio.com/docs/python/debugging

You'll probably have to pass rp2_main() (from rp2_main.py) as the entry point to the debugger.
